GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1846,8 +1873,8 @@ public static void main(String[] args) {
         Map<String, String> criteria = new HashMap<String, String>();
         Map<String, String> options = new HashMap<String, String>();
 //        options.put(TYPE_OPT, DATA_CLEANING_TYPE);
-        options.put(TYPE_OPT, DATA_ANALYSIS_TYPE);
-//        options.put(TYPE_OPT, COMPREHENSIVE_TYPE);
+//        options.put(TYPE_OPT, DATA_ANALYSIS_TYPE);
```

Comment:
why is this line commented out? If that code is not used maybe it should be deleted?

Reply:
This whole main() method is only for testing. So you can test the 3 different type by uncommenting one of them.